src/components/camera-tool.js

```diff
@@ -56,6 +56,9 @@ async function pixelsToPNG(pixels, width, height) {
   return new File([blob], "snap.png", { type: "image/png" });
 }
 
+// Don't show camera viewports on mobile VR to same framerate.
+const enableCameraViewport = !AFRAME.utils.device.isMobileVR();
```

Comment:
Might be nice to add a qs parameter or something here too. On desktop I am mildly annoyed when there is an abandoned camera in my view frustum that I know is doing a render I don't care about, but it may be that we should just be more careful about knowing when someone cares about seeing the camera preview. (e.g. perhaps there's a way for me to explicitly hide the preview, so that I don't have to destroy the camera when someone else wants it around and we can both get what we wanted)

Reply:
yeah this seems like something we should address at some point -- the preview is nice but we def show it too aggressively.